src/message/messageActionSheet.js

```diff
@@ -89,6 +90,10 @@ const editMessage = async ({ message, actions }: MessageAuthAndActions) => {
   actions.startEditMessage(message.id);
 };
 
+const deleteMessage = async ({ auth, message }: AuthAndMessageType) => {
+  patchMessage(auth, '(deleted)', message.id);
```

Comment:
Based on reading the backend code, I think we should be setting the content to `""`; the string `(deleted)` is what the backend replaces that with now, but we may change the markup in the future.

As a sidenote, we do support organization administrators completely deleting a message now, but that's a lower-priority feature that I don't think is worth working on for mobile yet.